packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/blob/BlobCollector.java

```diff
@@ -21,7 +21,7 @@ static void install(final ReactContext reactContext, final BlobModule blobModule
         new Runnable() {
           @Override
           public void run() {
-            JavaScriptContextHolder jsContext = reactContext.getJavaScriptContextHolder();
+            JavaScriptContextHolder jsContext = reactContext.getCatalystInstance().getJavaScriptContextHolder();
```

Comment:
This change might not be needed past https://github.com/facebook/react-native/pull/44360

Reply:
Yes, change updated

Reply:
@arushikesarwani94 Hi, PTAL again at the changes.

Reply:
cc: @cortinico